pro/cypress/e2e/features/adage.feature

```diff
@@ -0,0 +1,56 @@
+@P0
+Feature: Adage discovery
+
+  Background:
+    Given I go to adage login page with valid token
+
+  # note: ce scénario passe pas si on le met à la fin
+
+  Scenario: should put an offer in favorite
```

Comment:
pour la convention on met un it ou non ? (cette ligne n'est pas cohérente avec les autres)

Reply:
En effet, ajouté (et It en majuscule comme dans les autres features)